client/ayon_core/hosts/houdini/plugins/create/create_dynamic.py

```diff
@@ -64,11 +130,30 @@ def create(self, product_name, instance_data, pre_create_data):
         if custom_instance_data:
             instance_data.update(custom_instance_data)
 
+        instance_data["families"] = ["dynamic"]
+
         # TODO: Add support for multiple representations
-        files = pre_create_data["files"]
-        representations = [create_representation_data(files)]
+        files = pre_create_data.get("files", [])
+        if files:
+            representations = [create_representation_data(files)]
+
+        output_paths = pre_create_data.get("output_paths", [])
+        if output_paths:
```

Comment:
We should remove one of these two - have a single entry point that is used for the creator which we use as the "API". By exposing more ways we're just adding complexity ;)

By the way, `output_paths` really got me confused because I see these files actually as the 'input files' for the created instance (since basically this runtime instance expects the files to pre-exist hehe). Anyway, just `files` or `paths` or `filepaths` should suffice.